core/runtime/TRTEngine.cpp

```diff
@@ -128,14 +128,13 @@ TRTEngine::TRTEngine(
     for (size_t pyt_idx = 0; pyt_idx < outputs; pyt_idx++) {
       auto binding_name = _out_binding_names[pyt_idx];
       auto trt_idx = cuda_engine->getBindingIndex(binding_name.c_str());
-      std::string engine_binded_name = cuda_engine->getIOTensorName(inputs_size + pyt_idx);
```

Comment:
Original issue occurs here, since `trt_idx !=inputs_size + pyt_idx` in general (for example, when there are multiple output tensors from the TRT Engine)